.github/workflows/bitcoin-tests.yml

```diff
@@ -76,8 +76,10 @@ jobs:
           - tests::nakamoto_integrations::block_proposal_api_endpoint
           - tests::nakamoto_integrations::miner_writes_proposed_block_to_stackerdb
           - tests::nakamoto_integrations::correct_burn_outs
-          - tests::signer::stackerdb_dkg_sign
+          - tests::signer::stackerdb_dkg
+          - tests::signer::stackerdb_sign
           - tests::signer::stackerdb_block_proposal
+          - tests::signer::stackerdb_filter_bad_transactions
```

Comment:
`stackerdb_reward_cycle_transitions` used to be here, was it removed because it was flaking?

Reply:
I removed it because it actually should be "stackerdb_mine_2_nakamoto_reward_cycles" but I broke that test and am in the process of fixing it right now!

Reply:
Added back :)